tests/integration/DatabaseUpdateTest.php

```diff
@@ -107,18 +107,18 @@ public function testDataIsStillThereAfterColumnUpdate() {
 
 	public function testDataIsSetToDefaultAfterColumnUpdate() {
 		$schemaTo = $this->schemaManager->createSchema();
-		$schemaTo->getTable( 'action_log' )->dropColumn( 'al_timestamp' );
+		$schemaTo->getTable( 'backend_impressions' )->dropColumn( 'datetime' );
 		$this->updateDatabaseBySchema( $schemaTo );
 
-		$this->factory->getConnection()->insert( 'action_log', [ 'al_username' => 'test' ] );
+		$this->factory->getConnection()->insert( 'backend_impressions', [ 'banner_id' => 'banner1' ] );
 
 		$this->factory->newUpdater()->update();
 
-		$entity = $this->entityManager->getRepository( 'WMDE\Fundraising\Entities\ActionLog' )->findOneBy( [ ] );
+		$entity = $this->entityManager->getRepository( 'WMDE\Fundraising\Entities\BackendImpressions' )->findOneBy( [ ] );
```

Comment:
It does not actually matter here as there is only one record in the table but it could probably explicitly fetch the one with banner_id equal to `banner1`.